quickstart/known-limitations.md

```diff
@@ -4,10 +4,6 @@
 
 Once provisioned, neither Mayastor Disk Pools nor Mayastor Volumes can be re-sized. A Mayastor Pool can have only a single block device as a member. Mayastor Volumes are exclusively thick-provisioned.
 
-## Snapshots and Clones
-
-Mayastor has no snapshot or cloning capabilities.
```

Comment:
@tiagolobocastro @Abhinandan-Purkait should we add a limitation that "currently snapshots and restores are not supported on multi-replica volumes"?

Reply:
It's already on the snapshots section but I think it should also be here yes

Reply:
Yeah it's already a part of the snapshots section. Yeah it should be a part of limitations, as most folks visit the limitations sections before diving in.